cmd/xl-storage.go

```diff
@@ -247,6 +246,12 @@ func newXLStorage(ep Endpoint, cleanUp bool) (s *xlStorage, err error) {
 		diskIndex:  -1,
 	}
 
+	// We stagger listings only on HDDs.
+	if info.Rotational != nil && *info.Rotational {
```

Comment:
```suggestion
	if info.Rotational == nil || *info.Rotational {
```

Keeps current for all non-linux.